crates/core/transaction/src/view/transaction_perspective.rs

```diff
@@ -40,6 +40,12 @@ pub struct TransactionPerspective {
     pub denoms: asset::Cache,
     /// The transaction ID associated with this TransactionPerspective
     pub transaction_id: TransactionId,
+    // A map of base64-encoded nullfiers to the IDs of the transactions
+    // containing the outputs that they nullify.
+    pub transaction_ids_by_nullifier: HashMap<String, TransactionId>,
```

Comment:
should these fields also be getting populated in `ViewServer::transaction_info_by_hash` in the `penumbra-view` crate?

Reply:
Probably, but I figured I'd leave that up to core devs. These fields aren't a hard requirement — they're just _available_ (after this PR is merged) to use for generating richer views. We'll be populated these fields in the web app, and now the core devs can too if they choose to.

Reply:
Yeah, I think it's fine for the Rust view server to lag a bit, until there's a concrete use case (like a web frontend pulling from it).

Reply:
Filed #4083